test/integration/scene_broadcaster_system.cc

```diff
@@ -711,7 +711,8 @@ TEST_P(SceneBroadcasterTest, RemovedComponent)
   EXPECT_TRUE(received);
   EXPECT_TRUE(hasState);
 
-  // Run server again. The second time shouldn't send the state message.
+  // Run server again. The second time shouldn't have state info. The
+  // $message can still.
```

Comment:
This comment isn't clear to me. I think we're still expecting no message received, right?

Reply:
I think the changes to the sped up worlds causes the publish time to be triggered in the scene broadcaster. I'll clarify the message.

Reply:
Updated comments in 0734a60